components/ExploreTab.vue

```diff
@@ -0,0 +1,116 @@
+<script setup lang="ts"></script>
+<template>
+  <div class="cover-div">
+    <h1>Upcoming Book Fair</h1>
+    <img src="~/assets/images/books.jpeg" />
+    <button>Book</button>
+  </div>
+  <div class="popular-div">
+    <h2>Popular Now</h2>
+    <p>See All</p>
+  </div>
+  <div id="carouselExampleControls" class="carousel slide" data-bs-ride="carousel">
+    <div class="carousel-inner">
```

Comment:
You have used `div` element so many times. But you can you others html tag like <article>
<aside>
<main>
<details>
<summary>
<figure>
<dialog>